src/cli/argument-parser.ts

```diff
@@ -166,7 +166,8 @@ export default class CLIArgumentParser {
         this.experimental
             .allowUnknownOption()
             .option('--disable-multiple-windows', 'disable multiple windows mode')
-            .option('--experimental-compiler-service', 'run compiler in a separate process');
+            .option('--experimental-compiler-service', 'run compiler in a separate process')
+            .option('--cache', 'cache resources between test runs');
```

Comment:
```suggestion
            .option('--preserve-cache', 'preserve browser cache between test runs');
```

here and below

Reply:
In some tools, the `cache` option is used:
- GitLab (https://docs.gitlab.com/ee/ci/caching/)
- Jest (https://jestjs.io/docs/en/cli)
- npm (https://docs.npmjs.com/cli/v6/commands/npm-cache)

Reply:
> TestCafe doesn't use the browser cache. TestCafe has own cache and store some web application resources (script and styles) between test runs.

That makes a difference. `--cache` is good then

```suggestion
            .option('--cache', 'cache web assets between test runs');
```

Reply:
Ok.